### PR TITLE
Update es.xml with the new messages

### PR DESCRIPTION
**Pull request by Eudaldca**

### CODE REVIEW

**Comment by iEli2tyree011** [comment]
@Alfalik @Pablete1234 @TiTo-418 Feedback would be much appreciated :smile:

**Comment by Pablete1234** [comment]
@iEli2tyree011 everything seems fine to me

File: src/main/resources/lang/es.xml [hunk 1]
```diff
@@ -79,6 +80,12 @@
         <playableLeave>No puedes salir del terreno de juego</playableLeave>
         <playableInteract>No puedes interactuar con bloques de fuera del terreno de juego</playableInteract>
         <globalMuteEnabled>No puedes hablar en el chat mientras el silenciado global está habilitado.</globalMuteEnabled>
+        <teamAbsent>No estás un un equipo.</teamAbsent>
```
Comment (Alfalik) [inline src/main/resources/lang/es.xml]: No estás **en** un equipo.

File: src/main/resources/lang/es.xml [hunk 2]
```diff
@@ -121,6 +128,13 @@
         <autoBalance>Los equipos serán automáticamente balanceados si {0} continúa stackeado</autoBalance>
         <youChangedRating>Has cambiado tu puntuación para {1} {2} de {3} a {0}.</youChangedRating>
         <teamSizeChanged>{0} máxima capacidad establecida a {1}</teamSizeChanged>
+        <rankRemoved>Le has quitado el rango de {1} a {0}.</rankRemoved>
+        <ownRankRemoved>{0} te ha quitado tu rango de {1}.</ownRankRemoved>
+        <rankGiven>Le has dado el rango de {1} a {0}.</rankGiven>
+        <ownRankGiven>{0} te ha dado el rango de {1}.</ownRankGiven>
+        <noRanks>No hay rangos.</noRanks>
```
Comment (Alfalik) [inline src/main/resources/lang/es.xml]: No se han encontrado rangos.

File: src/main/resources/lang/es.xml [hunk 3]
```diff
@@ -37,6 +37,7 @@
         <noTeamMatch>No se ha encontrado ningún equipo con esos criterios.</noTeamMatch>
         <noRotMatch>No se ha encontrado ninguna rotación con esos criterios.</noRotMatch>
         <noPlayerMatch>Ningún jugador coincide con ese criterio.</noPlayerMatch>
+        <noRankMatch>Ningún rango coincide con esos criterios.</noRankMatch>
```
Comment (Alfalik) [inline src/main/resources/lang/es.xml]: Ningún rango coincide con los criterios ingresados.

**Comment by Alfalik** [comment]
I've did a quick review and almost everything seems okay except for 3 grammar errors. If you fix them, it could be merged.

**Comment by Eudaldca** [comment]
@Alfalik  How can I modify that file without creating a new commit? I still don't know how to use git further that git clone.

**Comment by CaptainElliott** [comment]
git commit --amend -m "new commit message here"

File: src/main/resources/lang/es.xml (hunk 3 above)
Comment (Pablete1234) [inline src/main/resources/lang/es.xml]: Ningún rango coincide con esos criterios.

**Comment by TiTo-418** [comment]
<a href="http://imgur.com/AymKbhE"></a>

**Comment by dentmaged** [comment]
Scrub @TiTo-418, you need to use quotes `"`.

**Comment by twizmwazin** [comment]
What elliott said, or `git commit --amend --no-edit` to keep the commit message

**Comment by TiTo-418** [comment]
@Eudaldca See above

**Comment by Eudaldca** [comment]
@TiTo-418 Yeah, I'd prefer to wait, I'll leave it to you and/or alfa, I have very little time now.

**Comment by TiTo-418** [comment]
@Eudaldca Ok, i will do it when the command branch is merged

**Pull request closed without merging**